### PR TITLE
Fix issue #664: seg fault when reading input file

### LINKED ISSUE #664: Segmentation fault in master reading the input file
When an input file is provided in the config file (via InputFile) and also provided in the command line, like this:

./SvtAv1EncApp -i Niklas_17fr_480x264_30fps_8bits_420.yuv -w 480 -h 264 -q 50 -fps-num 30 -fps-denom 1 -b Niklas_17fr_480x264_30fps_8bits_420_eo1_emode8_qp50.ivf -intra-period 31 -bit-depth 8 -c ../../Config/Sample.cfg -lad 17 -o recon_Niklas_17fr_480x264_30fps_8bits_420_eo1_emode8_qp50.yuv -enc-mode 8 -stat-report 1 -lp 2 -n 17

then, there is a segmentation fault:

thread #1, queue = 'com.apple.main-thread', stop reason = EXC_BAD_ACCESS (code=1, address=0x68)
    frame #0: 0x00007fff7bc7959c libsystem_c.dylib`flockfile + 18
libsystem_c.dylib`f

at 

SvtAv1EncApp`SetCfgInputFile(filename="input.yuv", cfg=0x000000010ae00430) at EbAppConfig.c:146:14

### PR DESCRIPTION
**Pull request by joelsole**
Closes #664

### CODE REVIEW

File: Source/App/EncApp/EbAppConfig.c [hunk 1]
```diff
@@ -139,6 +139,11 @@ static void SetCfgInputFile(const char *filename, EbConfig *cfg)
     else
         FOPEN(cfg->input_file, filename, "rb");
 
+    if (cfg->input_file == NULL) {
+        cfg->y4m_input = EB_FALSE;
```
Comment (mafonsoN) [inline Source/App/EncApp/EbAppConfig.c]: you might also want to set cfg->input_file_is_fifo to EB_FALSE

**Comment by joelsole** [comment]
Closes #664

**Comment by 1480c1** [comment]
> Closes #664

You need to put that in the original description (edit the op and add that in)

**Comment by joelsole** [comment]
Done. Thanks.

**Comment by mafonsoN** [comment]
Good. Let's merge this small change to fix master.

**Pull request merged**